src/System.CommandLine/CommandExtensions.cs

```diff
@@ -74,5 +74,13 @@ public static void ConfigureFrom(this Command command, IOptionBuilder optionBuil
             string commandLine,
             IReadOnlyCollection<char> delimiters = null) =>
             new Parser(command).Parse(commandLine);
+
+        public static Option AddOption<T>(this Command command, params string[] aliases)
```

Comment:
I think this deserves more discussion. I like that it's more terse, but it implies the existence of an `Option<T>`. Does that then imply there should be a `Command<T>`? It might be confusing to have to dig into the `Option<T>.Argument` property to do things like set the arity or add a validator. But then digging into the `Command.Argument` to set validators that deal with multiple options is already there and is confusing.

A potential complication is the introduction of multiple arguments of different types. I've been noodling on:

```cs
var x = new Command("move");
x.AddArgument(new Argument<DirectoryInfo[]>());
x.AddArgument(new Argument<DirectoryInfo>());
```

Would you want types like `Command<T1,T2>` or `Option<T1, T2>` to be possible?